app/src/main/java/org/covidwatch/android/data/diagnosiskeystoken/DiagnosisKeysToken.kt

```diff
@@ -4,13 +4,14 @@ import androidx.room.Entity
 import androidx.room.PrimaryKey
 import androidx.room.TypeConverters
 import org.covidwatch.android.data.CovidExposureConfiguration
+import org.covidwatch.android.data.NTPTime
 import org.covidwatch.android.data.converter.ExposureConfigurationConverter
 
 @Entity(tableName = "diagnosis_keys_token")
 @TypeConverters(ExposureConfigurationConverter::class)
 data class DiagnosisKeysToken(
     @PrimaryKey val token: String,
     val exposureConfiguration: CovidExposureConfiguration,
-    val providedTime: Long = System.currentTimeMillis(),
+    val providedTime: Long = NTPTime.currentTimeMillis(),
```

Comment:
We would to remove default value for this property and use injected `NtpTime` to use a proper time. But having here a proper world time is not critical, it's not even used anywhere for now. 

So we could leave the `System` time here because this class might be deleted in the near future when we update EN version.

Reply:
change reverted back to System call